pkgs/data/fonts/iosevka/bin.nix

```diff
@@ -1,19 +1,27 @@
-{ stdenv, fetchzip }:
+{ stdenv, fetchurl, unzip
+, variant ? ""
+}:
 
 let
-  version = "3.4.6";
-in fetchzip {
-  name = "iosevka-bin-${version}";
+  name = "iosevka" + stdenv.lib.optionalString (variant != "") "-" + variant;
+in stdenv.mkDerivation rec {
+  pname = "${name}-bin";
+  version = "3.7.1";
 
-  url = "https://github.com/be5invis/Iosevka/releases/download/v${version}/ttc-iosevka-${version}.zip";
+  src = fetchurl {
+    url = "https://github.com/be5invis/Iosevka/releases/download/v${version}/ttc-${name}-${version}.zip";
```

Comment:
I believe these are manual uploads to github? Just checking, because generated zips are unstable and have to use fetchzip

Reply:
These are, yes. They're all builds by the developer.